ibis/spark/tests/test_numeric.py

```diff
@@ -0,0 +1,15 @@
+import pytest
```

Comment:
This file should have some tests for inf/nan if they don't exist elsewhere.

Reply:
There's `test_isnan_isinf` in `ibis/tests/all/test_numeric.py`

Reply:
Cool.